[RHEL 7.9] kdump does not generate any report

Downstream report: https://bugzilla.redhat.com/show_bug.cgi?id=1816932

verify/rhel-7-9
Ooops, it happened again<hr>
```
# ----------------------------------------------------------------------
# testBasic (__main__.TestKdump)
#
Created symlink from /etc/systemd/system/multi-user.target.wants/kdump.service to /usr/lib/systemd/system/kdump.service.
Job for kdump.service failed because the control process exited with error code. See "systemctl status kdump.service" and "journalctl -xe" for details.

DevTools listening on ws://127.0.0.1:9666/devtools/browser/a355d766-bb53-434d-a994-0599c7726c7b
> info: %cDownload the React DevTools for a better development experience: https://fb.me/react-devtools font-weight:bold
> warning: grep: /sys/class/dmi/id/power/autosuspend_delay_ms: Input/output error

Generating grub configuration file ...
Found linux image: /boot/vmlinuz-3.10.0-1130.el7.x86_64
Found initrd image: /boot/initramfs-3.10.0-1130.el7.x86_64.img
Found linux image: /boot/vmlinuz-0-rescue-68948292bf234def81762309585741f3
Found initrd image: /boot/initramfs-0-rescue-68948292bf234def81762309585741f3.img
done
> warning: transport closed: disconnected
> log: transport closed, dropped message:  1:1!6
{"call":["/org/freedesktop/timedate1","org.freedesktop.DBus.Properties","Get",["org.freedesktop.timedate1","NTPSynchronized"]],"id":"5"}
> log: transport closed, dropped message:  1:1!6
{"call":["/org/freedesktop/timedate1","org.freedesktop.DBus.Properties","Get",["org.freedesktop.timedate1","NTPSynchronized"]],"id":"6"}
> log: transport closed, dropped message:  1:1!6
{"call":["/org/freedesktop/timedate1","org.freedesktop.DBus.Properties","Get",["org.freedesktop.timedate1","NTPSynchronized"]],"id":"7"}
> log: transport closed, dropped message:  1:1!6
{"call":["/org/freedesktop/timedate1","org.freedesktop.DBus.Properties","Get",["org.freedesktop.timedate1","NTPSynchronized"]],"id":"8"}
> log: transport closed, dropped message:  1:1!6
{"call":["/org/freedesktop/timedate1","org.freedesktop.DBus.Properties","Get",["org.freedesktop.timedate1","NTPSynchronized"]],"id":"9"}
> log: transport closed, dropped message:  1:1!6
{"call":["/org/freedesktop/timedate1","org.freedesktop.DBus.Properties","Get",["org.freedesktop.timedate1","NTPSynchronized"]],"id":"10"}
> info: %cDownload the React DevTools for a better development experience: https://fb.me/react-devtools font-weight:bold
# localhost:22 SSH-2.0-OpenSSH_7.4
# localhost:22 SSH-2.0-OpenSSH_7.4
# localhost:22 SSH-2.0-OpenSSH_7.4
> warning: grep: /sys/class/dmi/id/power/autosuspend_delay_ms: Input/output error

> warning: Unable to apply settings: nfs dump target isn't formated as server:path
> warning: Unable to apply settings: nfs dump target isn't formated as server:path
> warning: Unable to apply settings: Directory /var/crash2 isn't writable or doesn't exist.
> warning: transport closed: disconnected
Wrote screenshot to TestKdump-testBasic-rhel-7-9-127.0.0.2-2201-FAIL.png
Wrote HTML dump to TestKdump-testBasic-rhel-7-9-127.0.0.2-2201-FAIL.html
Wrote JS log to TestKdump-testBasic-rhel-7-9-127.0.0.2-2201-FAIL.js.log
Journal extracted to TestKdump-testBasic-rhel-7-9-127.0.0.2-2201-FAIL.log
> log: transport closed, dropped message:  1:1!6
{"call":["/org/freedesktop/timedate1","org.freedesktop.DBus.Properties","Get",["org.freedesktop.timedate1","NTPSynchronized"]],"id":"29"}
Traceback (most recent call last):
  File "test/verify/check-kdump", line 196, in testBasic
    m.execute("""find "{0}" -maxdepth 1 -mindepth 1 -type d -exec echo {{}} \;""".format(customPath)), "")
AssertionError: '' == ''

not ok 1 testBasic (__main__.TestKdump) # duration: 203s
```
First occurrence: 2020-03-25T09:09:10.079766
Times recorded: 1
Latest occurrences:

- 2020-03-25T09:09:10.079766

--- a/naughty/rhel-7/678-kdump-no-report
+++ b/naughty/rhel-7/678-kdump-no-report
@@ -1,4 +1,0 @@
-Traceback (most recent call last):
-  File "test/verify/check-kdump", line *, in testBasic
-    m.execute("""find "{0}" -maxdepth 1 -mindepth 1 -type d -exec echo {{}} \;""".format(customPath)), "")
-AssertionError: '' == ''

